source/about-mongosync.txt

```diff
@@ -157,6 +157,73 @@ If the reverse sync is successful, ``mongosync`` enters the ``RUNNING`` state.
 The synchronization continues in the reverse direction from the original 
 sync job. 
 
+.. _mongosync-considerations:
+
+Considerations for Continuous Sync
+----------------------------------
+
+Consider the following when using ``mongosync`` for continuous synchronization.
+
+Consistency 
+~~~~~~~~~~~
+
+For the destination cluster to have an up to date, consistent view of the 
+source cluster’s data, ``mongosync`` must :ref:`commit <c2c-api-commit>` . 
+If the source cluster shuts down before ``mongosync`` can commit, such as in 
+a disaster scenario, the destination cluster might not have a consistent 
+snapshot of the source data, and transactions might not appear atomically on 
```

Comment:
I don't think we need to mention transactions again.

I also think we need to be clear that the lack of consistent snapshot is because writes are applied in parallel and mongosync is eventually consistent.

The altering collection/index characteristics below should be in reference to other things that mongosync alters during a sync

Reply:
Gotcha. In that case, I'll split these up into different sections.